src/libcollections/binary_heap.rs

```diff
@@ -568,6 +568,9 @@ pub struct Iter <'a, T: 'a> {
     iter: slice::Iter<'a, T>,
 }
 
+unsafe impl<'a, T: Sync> Sync for Iter<'a, T> {}
+unsafe impl<'a, T: Sync> Send for Iter<'a, T> {}
```

Comment:
Currently it isn't. This can be removed, however, after `libcore` is properly `Sync` and `Send` audited.